.github/workflows/ci.yml

```diff
@@ -41,6 +41,6 @@ jobs:
       -
         name: integration-tests
         env:
-          SETUP_ENVTEST_VER: v0.0.0-20221214170741-69f093833822
-          ENVTEST_K8S_VERSION: 1.25
+          SETUP_ENVTEST_VER: v0.0.0-20240115093953-9e6e3b144a69
+          ENVTEST_K8S_VERSION: 1.28
```

Comment:
Why not 1.29 here? :thinking:

Reply:
It doesn't exist  :) 
Try

 ```
unset KUBEBUILDER_ASSETS
setup-envtest use --use-env -p path 1.29                                                                                                                                                                                                       
```

Reply:
A perfectly good explanation, indeed `setup-envtest` doesn't find any supported 1.29 version yet.